index.bs

```diff
@@ -1019,7 +1019,9 @@ partial interface Window {
 };
 </xmp>
 
-The <dfn attribute for=Window>cookieStore</dfn> getter steps are to return [=/this=]'s [=relevant settings object=]'s {{CookieStore}} object.
+A {{Window}} has an <dfn attribute for=Window>associated CookieStore</dfn>, which is a {{CookieStore}}.
+
+The <dfn attribute for=Window>cookieStore</dfn> getter steps are to return [=/this=]'s {{Window/associated CookieStore}}.
```

Comment:
And we definitely don't want this to be an IDL link `{{ ... }}`, I think just a regular `[=Window/associated CookieStore=]` link ?